xml/release-notes.xml

```diff
@@ -454,6 +454,22 @@
     </para>
    </sect2>
 
+   <sect2 xml:id="ibus-layout-name-change">
+    <title>Manual config migration of IBus is necessary due to layout name change</title>
+    <para>
+     Since IBus 1.5.23 renames some keyboard layouts, it cannot load configuration
+     containing these renamed layouts after upgrading. Thereby, it might change the
+     layout to US. The renamed layouts include be:*:ger, ch:*:ger, de:*:ger,
+     gr:*:gre, ro:*:rum, and, sk:*:slo.
```

Comment:
I am not very familiar with layout naming, how to understand something like `be:*:ger`? Does it mean that `be` was renamed to `ger`?

Reply:
IBus keyboard layout name consists of three parts (language, options, country) spitted by a colon. Here, the X:*:Y represents all the layout where language is X and country is Y. For example, this time de:*:ger layout was renamed to de:*:deu.

I know this representation is difficult to understand. I will try to find out natural names of those layouts.

Reply:
I see, thank you for the explanation! We could including your explanation in the text or we could create a table saying what has changed to what, if you think that's necessary.

Reply:
> three parts (language, options, country) spitted by a colon

Let me correct this explanation. The format is "${layout}:${variant}:${language}".

And through this discussion, I now understand what I have to do is just listing languages affected by this change. I will update later. Thanks.